engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -262,7 +262,7 @@ private void initRenderGraph() {
         BufferClearingNode readOnlyGBufferClearingNode = new BufferClearingNode(readOnlyGBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
         renderGraph.addNode(readOnlyGBufferClearingNode, "readOnlyGBufferClearingNode");
 
-        BufferClearingNode writeOnlyGBufferClearingNode = new BufferClearingNode(writeOnlyGBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+        BufferClearingNode writeOnlyGBufferClearingNode = new BufferClearingNode(writeOnlyGBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
         renderGraph.addNode(writeOnlyGBufferClearingNode, "writeOnlyGBufferClearingNode");
```

Comment:
Before this PR, the only one GBuffer was being cleared here, and the second was being cleared in some random Node in the process method() (don't even remember which buffer it was), along with a comment asking why it was necessary. I removed that comment and that glClear from that random Node, and shifted it to a proper BufferClearingNode to make things a lot more explicit. I also mentioned this specifically via comment in the commit that I made this change.